plugins/cmakebuilder-plugin/src/main/resources/hudson/plugins/cmake/CmakeBuilder/common-fields_zh_CN.properties

```diff
@@ -9,5 +9,5 @@ Build\ tool=\u6784\u5EFA\u5DE5\u5177
 Run\ build\ tool=\u8FD0\u884C\u6784\u5EFA\u5DE5\u5177
 Add\ build\ tool\ invocation=\u6DFB\u52A0\u6784\u5EFA\u5DE5\u5177
 
-CmakeBuilder.build_steps=\u6784\u5EFA\u6B65\u9AA4
-CmakeBuilder.actual_build_tool=make, ninja, nmake, ...
```

Comment:
Thanks to keeping track this. But there's no good back compatible solution. So I suggest that just keep the old one, then mark them.

Reply:
Yes, you will have to keep the old version (Run\ build\ tool=) for backward compatibility and
haave to ADD the new version (CmakeBuilder.build_steps=)

Reply:
Please also note that my pull request actually uses
`build_steps=` and
`actual_build_tool=` for the property names. 
The prefix `CmakeBuilder.` has been removed since it is already in the package name.

Reply:
If I understand correctly, we should keep `CmakeBuilder.build_steps` instead of removing them. Right? Plus, mark the old ones as stale.

Reply:
Yes. Keep both, marke the old ones as stale.